modules/setting/setting.go

```diff
@@ -737,6 +737,9 @@ func NewContext() {
 	}
 
 	if LFS.StartServer {
+		if LFS.ContentPath == "" {
```

Comment:
I think this needs something like:
```
	sec = Cfg.Section("lfs")
	LFS.ContentPath = sec.Key("CONTENT_PATH").MustString(path.Join(AppDataPath, "lfs"))
	if !filepath.IsAbs(LFS.ContentPath) {
		LFS.ContentPath = filepath.Join(workDir, LFS.ContentPath)
	}
```

Reply:
Yes, you're right. Need to check if absolute path. Think the first bit is ok tho?

Reply:
To be more consistent with other code I think it should be something like I gave example. Only thing I also have wrong for FS paths better is to use `filepath` methods not `path`

Reply:
Fair enough 👍 I have updated accordingly :)

Reply:
@cez81 did you forgot to do git push?

Reply:
Forgot to commit... :/